src/main/java/org/gitlab4j/api/models/MergeRequest.java

```diff
@@ -36,7 +36,11 @@ public class MergeRequest {
     private Date latestBuildStartedAt;
     private String mergeCommitSha;
     private String squashCommitSha;
+    // see https://docs.gitlab.com/ee/update/deprecations.html#merge_status-api-field
+    // -> deprecated since 15.6, use detailedMergeStatus instead
```

Comment:
I would use a javadoc comment, since compared to a regular comment, this is more helpful inside an IDE. Something like:

```suggestion
    /**
     * @deprecated since 15.6, use {@link #detailedMergeStatus} instead.
     * see https://docs.gitlab.com/ee/update/deprecations.html#merge_status-api-field
     */
```